README_EN.md

```diff
@@ -120,6 +120,8 @@ The Mini Sora open-source community is positioned as a community-driven initiati
 | 7) Video Generation Task in Papers With Code |  [Task](https://paperswithcode.com/task/video-generation) |
 | 8) Sora: A Review on Background, Technology, Limitations, and Opportunities of Large Vision Models |  [**Paper**](https://arxiv.org/abs/2402.17177), [Github](https://github.com/lichao-sun/SoraReview) |
 | 9) Open-Sora-Plan (PKU-YuanGroup) |  [Github](https://github.com/PKU-YuanGroup/Open-Sora-Plan) |
+| 10) State of the Art on Diffusion Models for Visual Computing | [Paper](http://arxiv.org/abs/2310.07204)  |
+| 11) Diffusion Models: A Comprehensive Survey of Methods and Applications | [**CSUR 24 Paper**](https://dl.acm.org/doi/10.1145/3626235), [Github](https://github.com/YangLing0818/Diffusion-Models-Papers-Survey-Taxonomy) |
```

Comment:
https://arxiv.org/abs/2209.00796

Reply:
arXiv version is better, please see https://github.com/mini-sora/minisora/issues/101#issuecomment-1972167603

Reply:
> arXiv version is better, please see [#101 (comment)](https://github.com/mini-sora/minisora/issues/101#issuecomment-1972167603)

Okay, I'll update it